assets/src/admin.js

```diff
@@ -50,14 +50,18 @@ jQuery(function ($) {
 	}
 
     // import log page
-    if ($('#tsml_import_log').length) {
-        const $filter = $('#filter_type');
-        const $log = $('#tsml_import_log');
-        // watch for filter change
-        $filter.on('change',(e) => {
+	const $filter = $('#filter_type');
+    if ($filter.length) {
+        $filter.on('change',() => {
             const type = $filter.val();
-            $log.find('tr').each((i,e) => $(e).toggle((!type && e.dataset?.type) || type === e.dataset?.type));
-        })
+            const url = new URL(window.location);
+			if (type) {
+				url.searchParams.set('filter_type', $filter.val());
+			} else if (url.searchParams.has('filter_type')) {
+				url.searchParams.delete('filter_type');
+			}
+			window.location = url.toString();
```

Comment:
append a query param (with reload) to be able to link to the page with the type selected

Reply:
Feel like the ideal set up is filter without reload _and_ persisting selections, but honestly 6 in one hand, half a dozen in the other.

Reply:
yeah - my thinking was to be able to link to the log page with a type seleted - i wasn't sure if i wanted the button on the import / export page to link to just the data source events or to the full log, or to have two buttons

anyway i'm sure we could do both probably as well. the reload didn't seem so bad though since you have to be at the top of the page to use it